src/lib/core/bidi/directionality.spec.ts

```diff
@@ -0,0 +1,120 @@
+import {async, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {Component, getDebugNode} from '@angular/core';
+import {Dir, Directionality} from './index';
+
+function initDir () {
+  document.documentElement.dir = '';
+  document.body.dir = '';
+}
```

Comment:
Call this something like `clearDocumentDirAttributes`? Also move to the bottom of the file, after the tests but before the test components